pallets/bridge/src/weights.rs

```diff
@@ -46,102 +46,27 @@ use frame_support::{
 use sp_std::marker::PhantomData;
 
 /// Weight functions needed for pallet_bridge.
-pub trait WeightInfo {
-    fn set_vote_threshold() -> Weight;
-    fn register_chain() -> Weight;
-    fn unregister_chain() -> Weight;
-    fn register_bridge_token() -> Weight;
-    fn unregister_bridge_token() -> Weight;
-    fn set_bridge_token_fee() -> Weight;
-    fn teleport() -> Weight;
-    fn materialize() -> Weight;
-}
+pub trait WeightInfo {	fn set_vote_threshold() -> Weight;	fn register_chain() -> Weight;	fn unregister_chain() -> Weight;	fn register_bridge_token() -> Weight;	fn unregister_bridge_token() -> Weight;	fn set_bridge_token_fee() -> Weight;	fn teleport() -> Weight;	fn materialize() -> Weight;}
 
 /// Weights for pallet_bridge using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
-impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-    fn set_vote_threshold() -> Weight {
-        (23_513_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(1 as Weight))
-            .saturating_add(T::DbWeight::get().writes(1 as Weight))
-    }
-    fn register_chain() -> Weight {
-        (24_264_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(1 as Weight))
-            .saturating_add(T::DbWeight::get().writes(1 as Weight))
-    }
-    fn unregister_chain() -> Weight {
-        (25_070_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(1 as Weight))
-            .saturating_add(T::DbWeight::get().writes(1 as Weight))
-    }
-    fn register_bridge_token() -> Weight {
-        (30_817_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(2 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-    fn unregister_bridge_token() -> Weight {
-        (30_536_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(1 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-    fn set_bridge_token_fee() -> Weight {
-        (31_773_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(2 as Weight))
-            .saturating_add(T::DbWeight::get().writes(1 as Weight))
-    }
-    fn teleport() -> Weight {
-        (101_743_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(4 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-    fn materialize() -> Weight {
-        (136_015_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(7 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-}
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {	fn set_vote_threshold() -> Weight {
+		(13_000_000 as Weight)			.saturating_add(T::DbWeight::get().reads(1 as Weight))			.saturating_add(T::DbWeight::get().writes(1 as Weight))	}	fn register_chain() -> Weight {
```

Comment:
should be formatted

Reply:
Something strange happens, these are formatted and make lint was passed 🤣